android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1426,4 +1426,25 @@ private static void logP3ARecords(
             }
         });
     }
+
+    /**
+     * Gets truncated address from a valid full contract address.
+     * @param address full contract address
+     * @return truncated address
+     */
+    @NonNull
+    public static String getTruncatedAddress(@NonNull final String address) {
+        if (address.isEmpty()) {
+            Log.w(TAG, "Empty contract address.");
```

Comment:
maybe we can put assert here to stop in Debug, otherwise that log in logcat could be easily missed as to me. The same about below Log

Reply:
Sure!